File: tests/fixtures/fixture_ts/src/lib/cache.ts
const store: Map<string, any> = new Map();

export function cacheGet(key: string): any {
  return store.get(key);
}

export function cacheSet(key: string, value: any, ttl: number = 3600): void {
  store.set(key, value);
}

export function cacheInvalidate(key: string): boolean {
  return store.delete(key);
}

File: tests/fixtures/fixture_python/app/utils/cache.py
_store: dict = {}

def cache_get(key: str):
    return _store.get(key)

def cache_set(key: str, value, ttl: int = 3600):
    _store[key] = value

def cache_invalidate(key: str) -> bool:
    return _store.pop(key, None) is not None
